
    PR ada/19942
    * utils.c (gnat_type_for_mode): Return null instead of ICE because
				    we asked for an unknown mode.

diff -u -r1.91 -r1.92
--- gcc/gcc/ada/utils.c	2005/02/12 20:02:35	1.91
+++ gcc/gcc/ada/utils.c	2005/02/13 20:08:29	1.92
@@ -1835,7 +1835,7 @@
   else if (SCALAR_INT_MODE_P (mode))
     return gnat_type_for_size (GET_MODE_BITSIZE (mode), unsignedp);
   else
-    gcc_unreachable ();
+    return NULL_TREE;
 }
 
 /* Return the unsigned version of a TYPE_NODE, a scalar type.  */
